h2o-py/tests/testdir_algos/stackedensemble/pyunit_stackedensemble_metalearner.py

```diff
@@ -66,7 +66,7 @@ def train_ensemble_using_metalearner(algo, expected_algo):
         if meta_params:
             assert(se.params['metalearner_nfolds']['actual'] == 3)
 
-        meta = h2o.get_model(se.metalearner()['name'])
+        meta = h2o.get_model(se.metalearner().model_id)
```

Comment:
unrelated to this PR, just to remove deprecation warning since the `metalearner()` is not a dictionary anymore and throws deprecation warning when using as dictionary.